pkgs/tools/misc/starship/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchFromGitHub, rustPlatform, openssl, pkgconfig, libiconv, darwin }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "starship";
+  version = "0.10.1";
+
+  src = fetchFromGitHub {
+    owner = "starship";
+    repo = "starship";
+    rev = "v${version}";
+    sha256 = "045lq4nsrdssmqbcj0551f2c5qd2rcvhs8gr4p4iniv7s89yz1xl";
+  };
+
+  buildInputs = [ openssl pkgconfig ] ++ stdenv.lib.optional stdenv.isDarwin [ libiconv darwin.apple_sdk.frameworks.Security ];
+
+  cargoSha256 = "126y8q19qr37wrj6x4hqh0v7nqr9yfrycwqfgdlaw6i33gb0qam9";
+
```

Comment:
I thought you still need to provide `platforms` with `rust`.

Reply:
I'm not sure how this comment ended up here... This comment was intended for the `meta` section.